content/en/docs/tasks/administer-cluster/kubeadm/rotate-certs.md

```diff
@@ -0,0 +1,117 @@
+---
+reviewers:
+- sig-cluster-lifecycle
+title: Certificate rotations under Kubeadm
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This page explains how to rotate the various certificates associated with a kubeadm k8s installation.
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+As part of upgrades, `kubeadm` will check all certificates it manages for expiration.
+If they are expiring within the next month, they will be replaced.
+
+Alternatively, the following commands can be used to granularly rotate individual certificates.
+
+{{< warning >}}
+
+Kubernetes CAs do not support OCSP.
+This means there is no mechanism for certificates to be invalidated before their expiration date.
+In the case of credential leaks, best practices are to create a new, uncompromised cluster.
+{{</ warning >}}
+
+## Cluster CA
+
+The cluster CA is initialised with a lifetime of [10 years][cacreation]. This is unlikely to require a reset.
+
+[ca creation]: https://github.com/kubernetes/kubernetes/blob/release-1.11/staging/src/k8s.io/client-go/util/cert/cert.go#L72
+
+## Kubelet client certificates
```

Comment:
@liztio might be I don't have all the info here, but why we need this if we are using Node certificate rotation via CSR auto-approved by the csrapprover controller?

Reply:
I need review here from a security engineer. My thinking was a compromised certificate could allow the compromise to continue if we used the same trust paths. But maybe that's not a real risk factor?